prow/github/client.go

```diff
@@ -408,6 +408,28 @@ func (c *Client) RemoveLabel(org, repo string, number int, label string) error {
 	return err
 }
 
+func (c *Client) AssignIssue(org, repo string, number int, logins []string) error {
+	c.log("AssignIssue", org, repo, number, logins)
+	_, err := c.request(&request{
+		method:      http.MethodPost,
+		path:        fmt.Sprintf("%s/repos/%s/%s/issues/%d/assignees", c.base, org, repo, number),
+		requestBody: map[string][]string{"assignees": logins},
```

Comment:
Github will not allow you to assign someone unless they are a member of the org.

Will make the change to expect `@rmmh` instead of just `rmmh`